sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -178,6 +187,98 @@ public virtual async Task<TrainingOperation> StartTrainingAsync(Uri trainingFile
 
         #endregion
 
+        #region Composed model
+
+        /// <summary>
+        /// Creates a composed model from a collection of existing trained models with labels.
+        /// </summary>
+        /// <param name="modelIds">List of model ids to use in the composed model.</param>
+        /// <param name="createComposedModelOptions">A set of options available for configuring the create composed model request.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>
+        /// <para>A <see cref="TrainingOperation"/> to wait on this long-running operation. Its <see cref="TrainingOperation.Value"/> upon successful
+        /// completion will contain meta-data about the trained model.</para>
+        /// <para>Even if operation fails, a model is created in the Form Recognizer account with an "invalid" status.
+        /// A <see cref="RequestFailedException"/> will be raised containing the modelId to access this invalid model.</para>
+        /// </returns>
+        public virtual TrainingOperation StartCreateComposedModel(IEnumerable<string> modelIds, CreateComposedModelOptions createComposedModelOptions = default, CancellationToken cancellationToken = default)
```

Comment:
I'm torn on using `TrainingOperation` or not. The implementation I need is exactly that but the name doesn't match.
I could create an identical class with a different name? or maybe a base class that inherits from `Operation` and then `Training` and `Composed` use it?
@kinelski  what do u think?

Reply:
I added a new base class `CreateCustomFormModelOperation`